File: wszystko/inf/python/funkcje.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  szblon.py

# zasieg zmiennych;lokalny, globalny
def dodaj(a, b):
    return a + b
    
def odejmij(a, b):
    return a - b
    
def podziel(a, b):
    return a / b
   
def pomnoz(a, b):
    return a * b

def main(args):
    a = int(input('Podaj 1. liczbę: '))
    print(a)
    b = int(input('Podaj 2. liczbę: '))
    print(b)
    
    
    print("Suma: ",  dodaj(a, b))
    print("Różnica: ", odejmij(a, b))
    print("Iloraz: ", podziel(a, b))
    print("Iloczyn: ", pomnoz(a, b))
    
    return 0
    
    # duck typing

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: rok 2018-2019/inf/cpp/funkcje.cpp
/*
 * szblon.cpp
 */


#include <iostream>

using namespace std;

int dodaj(int a, int b)
{
   return a + b;
}
int odejmij(int a, int b)
{
   return a - b;
}
int pomnoz(int a, int b)
{
   return a * b;
}
int podziel(int a, int b)
{
   return a / b;
}

int main(int argc, char **argv)
{
	float a, b;  // deklaracja zmiennej
    a = b = 0;  // inicjacja zmiennej
    
    cout << "Podaj 1. liczbę: ";
    cin >> a; 
    cout << a << endl; 
    
    cout << "Podaj 2. liczbę: ";
    cin >> b; 
    cout << b << endl;
    
    cout << "Suma: " << dodaj(a, b) << endl;
    cout << "Róznica: " << odejmij(a, b) << endl;
    cout << "Iloczyn: " << pomnoz(a, b) << endl;
    cout << "Iloraz: " << podziel(a, b) << endl;
    
	return 0;
}

